akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -1,154 +1,51 @@
-totally separate artifacts.
-
-A tightly coupled distributed application has served the industry and many Akka users well for years and is
-still a valid choice.
-
-### Distributed monolith
-
-There is also an anti-pattern that is sometimes called "distributed monolith". You have multiple services
-that are built and deployed independently from each other, but they have a tight coupling that makes this
-very risky, such as a shared cluster, shared code and dependencies for service API calls, or a shared
-database schema. There is a false sense of autonomy because of the physical separation of the code and
-deployment units, but you are likely to encounter problems because of changes in the implementation of
-one service leaking into the behavior of others. See Ben Christensen's
-[Don’t Build a Distributed Monolith](https://www.microservices.com/talks/dont-build-a-distributed-monolith/).
-
-Organizations that find themselves in this situation often react by trying to centrally coordinate deployment
-of multiple services, at which point you have lost the principal benefit of microservices while taking on
-the costs. You are in a halfway state with things that aren't really separable being built and deployed
-in a separate way. Some people do this, and some manage to make it work, but it's not something we would
-recommend and it needs to be carefully managed.
-
-## A Simple Cluster Example
+## Cluster API Extension
 
 The following configuration enables the `Cluster` extension to be used.
 It joins the cluster and an actor subscribes to cluster membership events and logs them.
 
-The `application.conf` configuration looks like this:
-
-```
-akka {
-  actor {
-    provider = "cluster"
-  }
-  remote.artery {
-    canonical {
-      hostname = "127.0.0.1"
-      port = 2551
-    }
-  }
-
-  cluster {
-    seed-nodes = [
-      "akka://ClusterSystem@127.0.0.1:2551",
-      "akka://ClusterSystem@127.0.0.1:2552"]
-
-    # auto downing is NOT safe for production deployments.
-    # you may want to use it during development, read more about it in the docs.
-    #
-    # auto-down-unreachable-after = 10s
-  }
-}
-
-```
-
-To enable cluster capabilities in your Akka project you should, at a minimum, add the @ref:[Remoting](remoting-artery.md)
-settings, but with `cluster`.
-The `akka.cluster.seed-nodes` should normally also be added to your `application.conf` file.
-
-@@@ note
-
-If you are running Akka in a Docker container or the nodes for some other reason have separate internal and
-external ip addresses you must configure remoting according to @ref:[Akka behind NAT or in a Docker container](remoting-artery.md#remote-configuration-nat-artery)
-
-@@@
-
-The seed nodes are configured contact points for initial, automatic, join of the cluster.
-
-Note that if you are going to start the nodes on different machines you need to specify the
-ip-addresses or host names of the machines in `application.conf` instead of `127.0.0.1`
```

Comment:
I think this information was lost from this page now. All from "To enable cluster..."

Reply:
and this would be good to have in typed/cluster.md too